examples/gcp_cli.py

```diff
@@ -97,3 +97,24 @@ def QueryLogs(args: 'argparse.Namespace') -> None:
   print('Found {0:d} log entries:'.format(len(results)))
   for line in results:
     print(json.dumps(line))
+
+def StartAnalysisVm(args: 'argparse.Namespace') -> None:
+  """Start forensic analysis VM.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  attach_disks = []
+  if args.attach_disks:
+    attach_disks = args.attach_disks.split(',')
```

Comment:
Same, make sure the variables here are as clean as possible before passing them on to `forensics.StartAnalysisVM`